src/app/clue.ts

```diff
@@ -206,6 +206,22 @@ export class Clue {
         // TODO: make-suggestion button
         // if state is 'suggest' OR if wasMovedToRoom is TRUE
 
+        if (this.player.state == this.player_state_suggest) {
+            // console.log(`enabling makeSuggestionButton`)
+            this.makeSuggestionButton.enable();
+        } else {
+            // console.log(`disabling make-suggestion button`)
+            this.makeSuggestionButton.disable();
+        }
+
+        if (suggester.setMovedToRoom(true))
```

Comment:
this doesn't make sense -- you're using a function from the backend code and assuming suggester variable is initialized somewhere but it is not.

Also note: the setMovedToRoom() function in the [backend repo](https://github.com/zbialik/clue-less) just sets the wasMovedToRoom attribute for a Player object to the provided input. It would not be used to determine the value of the attribute for the Player object.

Reply:
Thanks for the feedback and the fix, Zach. This is very helpful.